hugo/content/unlocked/build.md

```diff
@@ -64,7 +64,7 @@ make -C contrib run
 ```
 or use this podman command
 ```
-podman run --rm --mount type=bind,source="$(pwd)",target=/src -w /src -it ghcr.io/tillitis/tkey-builder:4 /usr/bin/bash
```

Comment:
While I agree about removing the tkey-builder number under devtools, I wonder if it is correct to do so under the unlocked pages. 

These instructions are for building a specific tag, as of now `TK1-24.03`, to be able to build it as we intended and infact released, the :4 needs to stay. Otherwise we may, implicitly, change the release build of an earlier tag when we release newer version of tkey-builder - a build that we may not have tested.